hcl2template/types.build.provisioners_test.go

```diff
@@ -0,0 +1,62 @@
+package hcl2template
+
+import (
+	"testing"
+
+	"github.com/hashicorp/hcl/v2"
+)
+
+func TestPackerConfig_ParseProvisionerBlock(t *testing.T) {
+	tests := []struct {
+		name        string
+		inputFile   string
+		expectError bool
```

Comment:
I think it would be good to pass in an expected error message, and just validate that we get the specific error we want.

Reply:
I agree, took that shortcut since I saw another test doing the same in the `hcl2template` directory, but you're right.

I'll add those before we can consider merging that PR

Reply:
Done in the latest reroll